marathon_lb.py

```diff
@@ -1602,6 +1623,13 @@ def get_arg_parser():
     parser.add_argument("--command", "-c",
                         help="If set, run this command to reload haproxy.",
                         default=None)
+    parser.add_argument("--retry-reload",
```

Comment:
infinite retries? that smells like it could hide problems. maybe this could indicate a "max retries before failure"?

Reply:
I'll change to `--max-reload-retries`